src/Microsoft.DotNet.InternalAbstractions/IDirectory.cs

```diff
@@ -11,13 +11,11 @@ internal interface IDirectory
 
         ITemporaryDirectory CreateTemporaryDirectory();
 
-        IEnumerable<string> EnumerateFiles(string path, string searchPattern);
+        IEnumerable<string> EnumerateAllFiles(string path);
```

Comment:
It's inconsistent to use **All** here but not in EnumerateFileSystemEntries below. We should either add All to both or keep 1:1 naming with System.IO.Directory and not have All in either.

Reply:
Hmm, actually i can just rename it to EnumerateFiles(string path). I changed the one with searchPattern overload since I find the only production code usage is passing a "*". And it is [the same](https://github.com/dotnet/corefx/blob/master/src/System.IO.FileSystem/src/System/IO/Directory.cs#L205) as just call `EnumerateFiles(string path)` overload.  cc @peterhuene

Reply:
Fixed, renamed and removed passing redundant "*"